src/PersonLink.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import classNames from 'classnames';
+import { useParams, Link } from 'react-router-dom';
+import { Person } from './types';
+
+const findParent = (
```

Comment:
The name of the `findParent` function does not fully convey the meaning of this function, because in addition to the search, we also return a reference to the parent, if it exists. Therefore, it would be better to name this function something like `getParentLink`.